src/SonarScanner.MSBuild.PreProcessor/Roslyn/RoslynAnalyzerProvider.cs

```diff
@@ -213,7 +210,10 @@ public IEnumerable<string> FetchAnalyzerAssemblies(IEnumerable<ActiveRule> activ
                     || !this.sqServerSettings.TryGetValue(PluginVersionPropertyKey(repoKey), out var pluginVersion)
                     || !this.sqServerSettings.TryGetValue(StaticResourceNamePropertyKey(repoKey), out var staticResourceName))
                 {
-                    this.logger.LogInfo(Resources.RAP_NoAssembliesForRepo, repoKey, language);
+                    if (!repoKey.StartsWith(SONARANALYZER_PARTIAL_REPO_KEY_PREFIX))
```

Comment:
As we have forced adding C# and VB.NET NuGets we should not warn when they are not found (when the plugin is not present on the server side).